docs/reference/elements/py-repl.md

```diff
@@ -46,3 +50,13 @@ The following will write "Hello! World!" to the div with id `replOutput`.
 ```
 
 Note that if we `print` from the REPL (or otherwise write to `sys.stdout`), the output will be printed in the [`py-terminal`](../plugins/py-terminal.md) if is enabled.
+
+### `<py-repl>` element with src
+Preload resource from src into the REPL
+```html
+<py-repl id="py-repl" output="replOutput" src="./src/py/Bubble Sort.py">
```

Comment:
Just a nitpick. Can we have a generic name instead of `Bubble Sort.py`?
A suggestion is `src="./src/py/py_code.py"`
Just so it's very clear to the user that's any code :)

Reply:
I shouldn't be operating it on my phone, please wait a minute, I'll fix it on my computer

Reply:
@marimeireles Thank you for waitingヾ(≧▽≦*)o

Reply:
haha
no problem at all, thank you so much for all the contributions!